CHANGELOG.md

```diff
@@ -8,23 +8,45 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## v0.15.0 [future]
 
 ### New Features
+
+#### surfaces.py
 - ToroidalSurface3D: line_intersections, linesegment_intersections, plane_intersections
-- OpenTriangleShell3D: triangle decimation
```

Comment:
Why deleting?

Reply:
It was an error. I believe I missed it when updating the file. Thanks, I'll add it again.